lib/active_model/serializer/adapter.rb

```diff
@@ -11,6 +13,7 @@ class Adapter
       autoload :Null
       autoload :FlattenJson
       autoload :CachedSerializer
+      include Configuration
```

Comment:
So, in https://github.com/rails-api/active_model_serializers/pull/1050/files#diff-ae1e38e96a8bf28ae37918a65c448c73R11 I just use e.g. `ActiveModel::Serializer.config.jsonapi_toplevel_meta = {}`  

the require could also be a problem since it's incompatible with rails autoload